### DISCUSSION TITLE
2.6.0

### DISCUSSION THREAD

**Discussion by magicbug** [magicbug/Cloudlog#2833] [General]
# Important

This version of Cloudlog introduces an updated frontend you will need to clean your cache this can be done by pressing **control + F5**

## Notable Changes

* Upgraded to Bootstrap 5
* Introduced new sub menus you can see these in action for Awards -> Grid Master Maps
* Introduced new wide themes (available via account settings)
* WAJA Award support added
* Improved QRZ Logbook handling
* Updated grid square handling to support 10 character grid squares (mostly for Microwave ops!)

## What's Changed
* E-Mail Bugfixes by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2715
* crlf and newline bugfix by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2717
* no need to press save twice by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2718
* Splitted VUCC into real VUCC and SAT-VUCC at Dashboard by @int2001 in https://github.com/magicbug/Cloudlog/pull/2716
* vucc link fix in german by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2719
* Fix SQL errors on migration by @phl0 in https://github.com/magicbug/Cloudlog/pull/2722
* Concat QSO IDs in PHP rather than SQL by @phl0 in https://github.com/magicbug/Cloudlog/pull/2720
* Use Ymd as file name to make sorting easier by @phl0 in https://github.com/magicbug/Cloudlog/pull/2723
* Load lib only if not loaded (prevent spilling of logs) by @phl0 in https://github.com/magicbug/Cloudlog/pull/2724
* Refactor LoTW imports by @phl0 in https://github.com/magicbug/Cloudlog/pull/2725
* Allow for callsign selection upon LoTW download by @phl0 in https://github.com/magicbug/Cloudlog/pull/2726
* Bootstrap 5 by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2707
* BS5 style improvements by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2729
* DXCC-Summary was not expandable due to BS5-Changes. by @int2001 in https://github.com/magicbug/Cloudlog/pull/2735
* RST in QSL Queue List by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2732
* Add user-selectable 'Name' column by @DJ3CE in https://github.com/magicbug/Cloudlog/pull/2734
* Quick lookup bs5 fix by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2738
* Russian translation update by @cats-shadow in https://github.com/magicbug/Cloudlog/pull/2737
* Clublog Realtime Upload (push to CL on QSO-Save) by @int2001 in https://github.com/magicbug/Cloudlog/pull/2739
* Fix missing grid lookups to handle lookups which do not return a grid (i.e. callsign not existing on hamqth/qrz) by @phl0 in https://github.com/magicbug/Cloudlog/pull/2745
* Update public OQRS for BS5 by @AndreasK79 in https://github.com/magicbug/Cloudlog/pull/2742
* Remove old maps by @phl0 in https://github.com/magicbug/Cloudlog/pull/2740
* [eQSL] Default QSLMSG by station profil (renew) by @abarrau in https://github.com/magicbug/Cloudlog/pull/2744
* Check if Band/Mode is provided.  by @int2001 in https://github.com/magicbug/Cloudlog/pull/2746
* Fix various errors in gridmap (in context of removal of old maps) by @phl0 in https://github.com/magicbug/Cloudlog/pull/2749
* Activated Grids were calculated by whole instance QSOs, not by owner by @int2001 in https://github.com/magicbug/Cloudlog/pull/2750
* [Awards] Added WAJA by @AndreasK79 in https://github.com/magicbug/Cloudlog/pull/2747
* Fix some issues in LoTW controller by @phl0 in https://github.com/magicbug/Cloudlog/pull/2752
* Delete QSO from Dashboard when pressing delete QSO by @int2001 in https://github.com/magicbug/Cloudlog/pull/2754
* Fix re-appending to LoTW download URL in loop by @phl0 in https://github.com/magicbug/Cloudlog/pull/2753
* Update and optimize some Simplified Chinese translations by @Lingluoluo in https://github.com/magicbug/Cloudlog/pull/2759
* We only need to load gridmap on maps page (not dashboard) by @phl0 in https://github.com/magicbug/Cloudlog/pull/2761
* [MAP] reduce highlight map line on dark theme by @abarrau in https://github.com/magicbug/Cloudlog/pull/2763
* [MAP Custom] custom qso plot (renew) by @abarrau in https://github.com/magicbug/Cloudlog/pull/2755
* Supressing Time at visitor-view (at the table there's date only too) by @int2001 in https://github.com/magicbug/Cloudlog/pull/2766
* [Awards] WAJA Map W.I.P. by @AndreasK79 in https://github.com/magicbug/Cloudlog/pull/2768
* [eQSL] fix for qslmsg with not accepted character by @abarrau in https://github.com/magicbug/Cloudlog/pull/2756
* Added Operator to Table/Quicksearch/Search by @int2001 in https://github.com/magicbug/Cloudlog/pull/2770
* Fix LBA Issue where POTA causes Problems by @int2001 in https://github.com/magicbug/Cloudlog/pull/2773
* Fix endless running loop for b0rken ADIF tags by @phl0 in https://github.com/magicbug/Cloudlog/pull/2774
* Version Dialog by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2769
* Re-sort JA prefectures by @phl0 in https://github.com/magicbug/Cloudlog/pull/2777
* BS5 fixes by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2780
* Fix Empty LBA by @int2001 in https://github.com/magicbug/Cloudlog/pull/2782
* Fixed Bug at "Show/Hide Version Info for all" by @int2001 in https://github.com/magicbug/Cloudlog/pull/2783
* Only reset timers in live QSO mode by @phl0 in https://github.com/magicbug/Cloudlog/pull/2785
* [QSO] fix lookup return result with the real callsign by @abarrau in https://github.com/magicbug/Cloudlog/pull/2779
* [QSO] Fix TimeOff on qso post by @abarrau in https://github.com/magicbug/Cloudlog/pull/2772
* Show version in dialog by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2787
* [Themes] Added my wide themes for Bootstrap 5 by @AndreasK79 in https://github.com/magicbug/Cloudlog/pull/2788
* Qslprint language by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2789
* fixed bad thead in wide theme by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2793
* [BS5] SimpleFLE and Tables by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2795
* one line too much by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2796
* Fix gridmap plotting re visitor parameter by @phl0 in https://github.com/magicbug/Cloudlog/pull/2797
* Download QRZ-Confirmations to Cloudlog by @int2001 in https://github.com/magicbug/Cloudlog/pull/2792
* Migscript to extend the default-qsl-col at users for qrz by @int2001 in https://github.com/magicbug/Cloudlog/pull/2799
* Removed COL_VUCC_GRID from migscript by @int2001 in https://github.com/magicbug/Cloudlog/pull/2800
* down function was missing by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2801
* Submenu bs5 by @AndreasK79 in https://github.com/magicbug/Cloudlog/pull/2808
* Catch empty QRZ.com response when downloading qrz-confirmations by @int2001 in https://github.com/magicbug/Cloudlog/pull/2807
* Contest View fix by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2805
* [BS5] Fixing some style issues by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2798
* [MAP] highlight ligne remove on all map  by @abarrau in https://github.com/magicbug/Cloudlog/pull/2803
* Check file/dir permissions prior to LoTW download by @phl0 in https://github.com/magicbug/Cloudlog/pull/2806
* Award QSO Data by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2813
* Check for empty LoTW/eQSL dates and catch the error by @phl0 in https://github.com/magicbug/Cloudlog/pull/2812
* WIP: Generalize gridmaster functions by @phl0 in https://github.com/magicbug/Cloudlog/pull/2816
* Search/Quicklog on Mobile by @HB9HIL in https://github.com/magicbug/Cloudlog/pull/2817
* Add DL Gridmaster by @phl0 in https://github.com/magicbug/Cloudlog/pull/2819
* LX Gridmaster by @phl0 in https://github.com/magicbug/Cloudlog/pull/2820
* New was map by @AndreasK79 in https://github.com/magicbug/Cloudlog/pull/2818
* Naming of subdivisions by @phl0 in https://github.com/magicbug/Cloudlog/pull/2771
* Ten digit grids by @phl0 in https://github.com/magicbug/Cloudlog/pull/2821
* Prevent empty Call on editing or logging via API by @int2001 in https://github.com/magicbug/Cloudlog/pull/2823
* Qrz key check fix by @int2001 in https://github.com/magicbug/Cloudlog/pull/2826
* Qra whitespace fix by @int2001 in https://github.com/magicbug/Cloudlog/pull/2828
* Qslprintfix by @int2001 in https://github.com/magicbug/Cloudlog/pull/2831
* Heal some strange behaviours of qrz-qsls by @int2001 in https://github.com/magicbug/Cloudlog/pull/2830
* Tag 2.6 by @magicbug in https://github.com/magicbug/Cloudlog/pull/2832

## New Contributors
* @DJ3CE made their first contribution in https://github.com/magicbug/Cloudlog/pull/2734

**Full Changelog**: https://github.com/magicbug/Cloudlog/compare/2.5.2...2.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/magicbug/Cloudlog/releases/tag/2.6.0'>2.6.0</a>.</em>

**Comment by substa** [comment]
Thank you for this nice release. When will the docker image be updated?

**Reply by HB9HIL** [reply]
@substa It was decided to drop the support for Docker. So there wont be a new release on docker. Please make a proper Backup of your Log and install Cloudlog the traditional way.

**Reply by magicbug** [reply]
As @HB9HIL says I have decided to restore some of my time for development I have dropped supplying a docker file as the support burden had become too much, that said if someone wants to fork Cloudlog and supply a docker image I'm completely OK with it.

I appreciate this will affect some users.

**Reply by rayslinky** [reply]
That's a shame. docker is a handy way of running a bunch of disparate software on a single host.

**Reply by substa** [reply]
Ok, I had moved everything from a classic installation to docker just a few weeks ago, because I think it's definitely easier to maintain and that's how I run all the other self-hosted services on my machines. Going back to the old way of managing the various services I think is a step backwards, but of course I accept the decision of the maintainers.
Thanks always for this great software, 73

**Reply by bgritzmacher** [reply]
I pushed an image to my own repository for 2.6.0 using the same Dockerfile that was present in 2.5.2: https://hub.docker.com/r/mervburger/cloudlog

It *should* just be a drop-in replacement for the original, with all the same caveats for updating before on the old images, otherwise I guarantee nothing. All I care is that I was able to switch to it and it works for me 😉

If you switch your docker config to it, make sure you back up, etc.

Builds will be manually run, so they won't be in perfect step with releases here, but I'll keep an eye out for new releases and make an updated build when possible.

@magicbug I've put in language in my description to clearly state that my image is unofficial, and that you do not provide support for it, let me know if I should change anything there.
